packages/apollo-server-azure-functions/src/azureFunctionApollo.ts

```diff
@@ -11,16 +11,16 @@ import {
 import { Headers, ValueOrPromise } from 'apollo-server-env';
 
 export interface AzureFunctionGraphQLOptionsFunction {
-  (request: FunctionRequest, context: HttpContext): ValueOrPromise<
+  (request: HttpRequest, context: Context): ValueOrPromise<
     GraphQLOptions
   >;
 }
 
 export interface AzureFunctionHandler {
   (
-    context: HttpContext,
-    request: FunctionRequest,
-    callback: (err?: any, output?: FunctionResponse) => void,
+    context: Context,
```

Comment:
Something isn't quite right with this new `Context` type, or something is off with the way that we currently call `context.done` for a normal (read: non-error) response.

I'm under the impression, [based on these Azure docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node#contextdone-method) a pattern would be to pass `null` in the error parameter if there is no error.  The previous version of this type (provided in the `azureFunctions.ts`'s `Context` type which this PR removes) offered a `done` which allowed that `null` value, but is now producing errors as that's not a supported signature in [`@azure/functions`'s typings](https://github.com/Azure/azure-functions-nodejs-worker/blob/800a06eb2f805a84a8fdc21d97c0b869913ba28e/types/public/Interfaces.d.ts#L58).

My suspicion is that perhaps we need to change the way we invoke `context.done`, but I'm not clear on why or when that changed (or if that's the desired move!).

Reply:
Yeah, this is a really good point. In testing, I've found that passing an empty string (`""`) worked the same as `null` in TypeScript; passing anything else breaks everything. 

For this PR, I've changed `null` to be `""` for now. 
For the future, I've [opened a PR against the `@azure/functions` typings](https://github.com/Azure/azure-functions-nodejs-worker/pull/194) asking for guidance on this.

@abernix let me know if you think there is anything else we should do to cover the bases on this.

Reply:
@michael-watson 	Out of curiosity, what does passing `undefined` do?

Reply:
It does work as well, but the Azure Functions team [agreed with your assessment](https://github.com/Azure/azure-functions-nodejs-worker/pull/195) on the docs and [I took all the credit](https://github.com/Azure/azure-functions-nodejs-worker/pull/195#issue-264631390).

I've updated our implementation to `null` as I think semantically, `undefined` doesn't make sense because it is defined. It's defined as no error, or `null`. But I guess it really doesn't matter though, both are acceptable responses now.